projects/android/webrtc-native/src/main/cpp/jni_onload.cpp

```diff
@@ -0,0 +1,26 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// See LICENSE in the project root for license information.
```

Comment:
New shorter license header for open-source projects is:
> // Copyright (c) Microsoft Corporation.
// Licensed under the MIT License.